src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -216,9 +245,12 @@ uct_rc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface, void *arg,
     ucs_status_t       status;
 
     ucs_assert(ep != NULL);
-    uct_rc_mlx5_common_update_tx_res(iface, &ep->tx.wq, &ep->super.txqp, pi);
     uct_rc_txqp_purge_outstanding(iface, &ep->super.txqp, ep_status, pi, 0);
 
+    /* Don't need to invoke UCT pending requests for a given UCT EP */
```

Comment:
/* Do not invoke pending requests on a failed endpoint */

Reply:
fixed